src/scrubber.ts

```diff
@@ -41,7 +54,8 @@ export class Scrubber {
 
         // Deep traverse
         if (typeof dataCopy[key] === 'object' && dataCopy[key]) {
-          dataCopy[key] = this.applyScrubbers(dataCopy[key])
+          const parentsNext = parents ? [...parents, key] : [key]
```

Comment:
You can do (feel free to ignore):

```ts
parents = [...parents, ley]
```

And in the function definition you can default parents to `[]` as `parents: string[] = []`.